_artifacts/document-attributes.adoc

```diff
@@ -96,7 +96,7 @@ endif::OP[]
 :EAP: Red Hat JBoss EAP
 :EAP_LONG: Red Hat JBoss Enterprise Application Platform
 :EAP_VERSION: 7.3
-:EAP_VERSION_LONG: 7.3.4
+:EAP_VERSION_LONG: 7.3.0
```

Comment:
[technical] If `EAP_VERSION_LONG` minor version should be 7.3.0 rather than 7.3.4 then this change needs to be backported to 7.10.x and the docs republished. `AMQ_BROKER_VERSION_LONG` does not need to be backported.

Reply:
@ksuta is Clifton correct? Should I back port this change to 7.10?

Reply:
@emmurphy1 @Ksuta What is the supported EAP version for 7.11.x? Is it perhaps a typo and it should be 7.4.0 for `EAP_VERSION_LONG` for 7.11.x?

Reply:
@csherrar, no 7.3.0 is correct and I will updated in the 7.10 docs.